cfgov/jinja2/v1/ask-cfpb/see-all.html

```diff
@@ -11,18 +11,23 @@ <h1>
       {{ page.title }}
     </h1>
     {% if page.glossary_terms %}
-        <div class="terms">
+        <dl class="terms" itemscope itemtype="http://schema.org/DefinedTermSet">
         {% for term in page.glossary_terms %}
             <div class="term {{'' if loop.index == 1 else 'block block__border-top block__padded-top'}}"
-                 id="{{ term.anchor(page.language) }}">
-            <h2>{{ term.name(page.language) }}</h2>
-                {{ term.definition(page.language) | richtext }}
-                {% if term.answer_page_en %}
-                    <p class="term-link"><a href="{{ term.answer_page_url(page.language) }}"> {{ _('Read more') }} </a></p>
-                {% endif %}
+                 id="{{ term.anchor(page.language) }}"
+                 itemscope
+                 itemtype="http://schema.org/DefinedTerm">
+                <link itemprop="url" href="{{ page.full_url }}#{{ term.anchor(page.language) }}">
+                <dt class="term_name" itemprop="name">{{ term.name(page.language) }}</dt>
+                <dd class="term_definition" itemprop="description">
+                    {{ term.definition(page.language) | richtext }}
+                    {% if term.answer_page_en -%}
+                        <a class="term_link" href="{{ term.answer_page_url(page.language) }}">{{ _('Read more') }}</a>
```

Comment:
I don't think we need to worry about this for this first pass, but I wonder how we should treat this link in the DefinedTerm schema. It doesn't quite feel like part of the `description` to me, but I'm not sure what else to do with it. I'm fine leaving it for now and refactoring later if needed.

Reply:
Hmm... I see what you mean, but I don't think that there is a good alternative that's immediately obvious.

Reply:
Agreed. I'm good leaving this as-is for now.